### ISSUE TITLE
Spring.Data.NHibernate4 shows dependency on NHibernate 3.3.1.4.000 using dotPeek

### ISSUE THREAD

**Issue by ynauls** [spring-projects/spring-net#107]
I am running into an error when programmatically loading an assembly that references Spring.Data.NHibernate4,

```
ERROR 
Could not load file or assembly 'NHibernate, Version=3.3.1.4000, Culture=neutral, PublicKeyToken=aa95f207798dfdb4' or one of its dependencies. The system cannot find the file specified.
```
Using JetBrains dotPeek shows NHibernate 3.3.1.4.0000 as a dependency for Spring.Data.NHibernate4

**Comment by lahma** [comment]
Well that went fine then... This will be fixed in patch release in the future, but in the interim you can make sure you have latest NHibernate 4 NuGet package installed (4.0.3) and create assembly redirect for NHibernate 4. It should work fine with something like...

```xml
<configuration>

...

  <runtime>
    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
      <dependentAssembly>
        <assemblyIdentity name="NHibernate" publicKeyToken="aa95f207798dfdb4" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-4.0.3.4000" newVersion="4.0.3.4000" />
      </dependentAssembly>
    </assemblyBinding>
  </runtime>      
  
...  

</configuration>
```

**Comment by sbohlen** [comment]
Fixed in 2.0.1-GA just pushed to NuGet.

**Comment by ynauls** [comment]
Thank you @sbohlen
